test/integration/targets/exos_vlans/tests/httpapi/merged.yaml

```diff
@@ -0,0 +1,46 @@
+---
+- debug:
+    msg: "START exos_vlans merged integration tests on connection = {{ ansible_connection }}"
+
+- include_tasks: _reset_config.yaml
+
+- include_tasks: _populate_config.yaml
+
+- block:
+  - name: Merge the provided configuration with the existing running configuration
+    exos_vlans: &merged
+      config:
+        - name: vlan_30
```

Comment:
Add some vlans whose state will be suspended in the merged, overridden, replaced tests.